### PR TITLE
Fix typo "an other attempt" => "another attempt"

### PR DESCRIPTION
**Pull request by sirreal**
Fixes a small typo in Jetpack connect.

> Error authorizing. Page is refreshing for an other attempt.

becomes

> Error authorizing. Page is refreshing for another attempt.

### CODE REVIEW

File: client/signup/jetpack-connect/jetpack-connect-notices.jsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export default React.createClass( {
 			return noticeValues;
 		}
 		if ( this.props.noticeType === 'retryingAuth' ) {
-			noticeValues.text = this.translate( 'Error authorizing. Page is refreshing for an other attempt.' );
+			noticeValues.text = this.translate( 'Error authorizing. Page is refreshing for another attempt.' );
```
Comment (a8ci18n) [inline client/signup/jetpack-connect/jetpack-connect-notices.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

**Comment by yoavf** [comment]
LGTM

**Comment by sirreal** [comment]
If those CircleCI linter tests need to pass, I'd prefer to roll this into #12991

**Comment by sirreal** [comment]
Rolled into #12991 at 408332a579d2b2bb5dd280916ba7d87cd00ddf99

**Pull request closed without merging**